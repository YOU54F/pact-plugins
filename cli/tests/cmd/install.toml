bin.name = "pact-plugin-cli"
args = "install --help"
