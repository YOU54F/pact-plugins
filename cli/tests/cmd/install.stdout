pact-plugin-cli-install [CLIVERSION]
Install a plugin

USAGE:
    pact-plugin-cli install [OPTIONS] <SOURCE>

ARGS:
    <SOURCE>
            Where to fetch the plugin files from. This should be a URL

OPTIONS:
    -h, --help
            Print help information

    -t, --source-type <SOURCE_TYPE>
            The type of source to fetch the plugin files from. Will default to Github releases.
            
            Valid values: github

    -V, --version
            Print version information

    -y, --yes
            Automatically answer Yes for all prompts
